changelog/unreleased/make-all-paths-configurable.md

```diff
@@ -0,0 +1,5 @@
+Change: Make all paths configurable and default to `/var/tmp/ocis/<service>/...`
+
+Aligned all services to use a subdir of `/var/tmp/ocis/` by default. Also made some missing temp paths configurable via env vars and config flags.
+
+https://github.com/owncloud/ocis/pulls/1080
```

Comment:
What is wrong with the change log? Does it really require an issue?!?

Reply:
My guess is that using backticks might be problem, specially in line 1. That ends with a backtick, and maybe there is an edge-case problem with that sort of thing?

Reply:
I'd guess the same. The structure of the changelog looks ok - a primary URL is required, but it can be any URL (issue, PR, third party resource, ...).

Reply:
had  `.../pulls/1080` instead of `.../pull/1080` in it ... which really does not exist ... 😞